modules/carto/src/layers/heatmap.ts

```diff
@@ -58,6 +56,8 @@ vec3 colorGradient(float value) {
 }
```

Comment:
Not new in this PR, but on lines 49–52, I'm concerned that color ranges all go to white at high weights, even when the color palette does not include white. The behavior differs from the documentation:

> Pixels with weight more than `maxValue` are capped to the last color in `colorRange`.

In particular when the basemap itself is white, it seems important that the user should be able to select a different color to represent maximum weight. Is this intended?